android/src/com/mapswithme/maps/MwmApplication.java

```diff
@@ -109,7 +108,10 @@ public synchronized void initNativeCore()
     initPaths();
     nativeInit(getApkPath(), getDataStoragePath(), getTempPath(), getObbGooglePath(),
                BuildConfig.FLAVOR, BuildConfig.BUILD_TYPE,
-               Yota.isFirstYota(), UiUtils.isSmallTablet() || UiUtils.isBigTablet());
+               UiUtils.isSmallTablet() || UiUtils.isBigTablet());
+
+    nativeStart(findMapStoragePath());
```

Comment:
findMapStoragePath() relies on the results of nativeInit(). Specifically on value of "StoragePath" saved to settings.ini.